src/com/epam/javacourse/hometask3/Book.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.javacourse.hometask3;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public final class Book extends Bookstore {
+    private final String isbn;
+    private final String name;
+    private final Author authors;
+    private final Publisher publisher;
+    private final String publicationDate;
+    private final boolean isPaperback;
+
+    public Book(String isbn, String name, Author authors, Publisher publisher, String publicationDate,
+                boolean isPaperback) {
+        this.isbn = isbn;
+        this.name = name;
+        this.authors = authors;
+        this.publisher = publisher;
+        this.publicationDate = publicationDate;
+        this.isPaperback = isPaperback;
+    }
+
+    public Author[] getAuthors() {
+        return new Author[]{authors};
```

Comment:
What is the purpose of using an array here?
As I've noticed each book has only one author in your book set. And you pass a single author in the `Book()` constructor.